miio/vacuum.py

```diff
@@ -110,10 +110,10 @@ def consumable_status(self) -> ConsumableStatus:
         """Return information about consumables."""
         return ConsumableStatus(self.send("get_consumable")[0])
 
-    def consumable_reset(self):
+    def consumable_reset(self, name):
         """Reset consumable information."""
-        raise NotImplementedError("unknown parameters")
-        # self.send("reset_consumable", ["unknown"])
+        # name = ["main_brush_work_time", "side_brush_work_time", "sensor_dirty_time" or "filter_work_time"]
```

Comment:
Could you create a `Consumable` enum, and use it instead of passing strings around. See https://github.com/rytilahti/python-miio/blob/master/miio/vacuum.py#L20 (and its use in https://github.com/rytilahti/python-miio/blob/master/miio/vacuum.py#L164) for example.

We do not need to follow their naming scheme either, I think it makes sense here to go for the same naming as in `ConsumableStatus` (https://github.com/rytilahti/python-miio/blob/master/miio/vacuumcontainers.py#L284).